pkgs/development/python-modules/pytest-sanic/default.nix

```diff
@@ -47,5 +47,6 @@ buildPythonPackage rec {
     homepage = "https://github.com/yunstanford/pytest-sanic/";
     license = licenses.asl20;
     maintainers = with maintainers; [ costrouc ];
+    broken = true; # 2021-11-04
```

Comment:
This package is not dependency of any other package now that `sanic` itself doesn't depend on it.

It is a good candidate for removal.